mkosi/__init__.py

```diff
@@ -234,21 +234,22 @@ def check_root_populated(context: Context) -> None:
 
 def configure_os_release(context: Context) -> None:
     """Write IMAGE_ID and IMAGE_VERSION to /usr/lib/os-release in the image."""
-    if not context.config.image_id and not context.config.image_version:
+    if not any ([context.config.image_id, context.config.image_version, context.config.hostname]):
```

Comment:
No space between name and argument
```suggestion
    if not any([context.config.image_id, context.config.image_version, context.config.hostname]):
```

Though I'd prefer
```suggestion
    if not (context.config.image_id or context.config.image_version or context.config.hostname):
```
since it's still shorter and less visual clutter (commas and braces).